### PR TITLE
feat: add router settings for LiteLLM

### PR DESCRIPTION
**Pull request by rodonguyen**
- Add router settings for LiteLLM

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -161,3 +161,6 @@ cython_debug/
 
 bin
 local_sandbox
+litellm_uuid.txt
+
+
```
Comment (samhwang) [inline .gitignore:166]: - delete the extra lines, only keep 1
- can we have an explanation why this file is added here?

```suggestion
# Reason why this file is added
litellm_uuid.txt
```

**Review by samhwang — CHANGES_REQUESTED** [review]

File: src/litellm/proxy_config.yaml [hunk 2]
```diff
@@ -10,3 +10,11 @@ model_list:
       model: ollama/phi
 litellm_params:
   drop_params: True
+
+litellm_params:
+  drop_params: True
+
+router_settings:
+  routing_strategy: least-busy      # Literal ["simple-shuffle", "least-busy", "usage-based-routing", "latency-based-routing"]
```
Comment (nqngo) [inline src/litellm/proxy_config.yaml:18]: Remove comments. Comment should be used to explain the codes. For everything else there is git.

**Review by nqngo — DISMISSED** [review]

File: src/litellm/proxy_config.yaml [hunk 3]
```diff
@@ -10,3 +10,10 @@ model_list:
       model: ollama/phi
 litellm_params:
   drop_params: True
+
+litellm_params:
```
Comment (samhwang) [inline src/litellm/proxy_config.yaml:14]: duplicate?

File: .gitignore [hunk 4]
```diff
@@ -152,12 +133,7 @@ dmypy.json
...
-#  JetBrains specific template is maintained in a separate JetBrains.gitignore that can
-#  be found at https://github.com/github/gitignore/blob/main/Global/JetBrains.gitignore
-#  and can be added to the global gitignore or merged into this file.  For a more nuclear
-#  option (not recommended) you can uncomment the following to ignore the entire idea folder.
-#.idea/
-
 bin
 local_sandbox
+# litellm auto generated files
+litellm_uuid.txt
```
Comment (samhwang) [inline .gitignore:139]: add an extra line before each section

```suggestion

# litellm auto generated files
litellm_uuid.txt
```

**Review by samhwang — CHANGES_REQUESTED** [review]

File: src/litellm/proxy_config.yaml (hunk 3 above)
Comment (rodonguyen) [reply src/litellm/proxy_config.yaml:14]: oops don't know why it happened

**Review by samhwang — APPROVED** [review]

**Pull request merged**